### ISSUE TITLE
[@hono/vite-dev-server] Best way to develop locally?

### ISSUE THREAD

**Issue by kevbook** [honojs/vite-plugins#42]
I'm using a basic `vite-dev-server` setup (see below). `abc` gets caught within the vite process and vite displays the error and watches the file system and restarts the server. But `outside` doesn't get caught the same way, and the vite process crashes. What is the recommended way to run a local dev server (via nodemon?)

```ts
export default defineConfig({
  plugins: [
    devServer({
      entry: 'src/index.ts', // The file path of your application.
    }),
  ],
});
```

```js 
import { Hono } from 'hono';
const app = new Hono();
outside;
app.get('/', (c) => {
  abc;  
  return c.text('Hello Vitess!');
});
export default app;
```

**Comment by yusukebe** [comment]
Hi @kevbook 

Maybe we should make it does not crash and restart automatically also in that case. I'll take a look later.

> What is the recommended way to run a local dev server (via nodemon?)

This local dev server can run only Vite. So, for instance, you can use `tsx watch src/index.ts` command if you run it on Node.js.

**Comment by kevbook** [comment]
Yea, it would be awesome if you could make it not crash. I use Sveltekit (uses Vite) which has a similar dev experience.

**Comment by yusukebe** [comment]
> So, for instance, you can use `tsx watch src/index.ts` command if you run it on Node.js.

Ah, it also crashes with using `tsx`.

>  I use Sveltekit (uses Vite) which has a similar dev experience.

Okay, I can refer to it. Thanks!

**Comment by yusukebe** [comment]
@kevbook 

I've released `v0.3.2` which does not crash in that case. Please try it out!
